projects/ccd-case-ui-toolkit/src/lib/shared/components/search-result/search-result.component.html

```diff
@@ -61,6 +62,7 @@ <h2 class="heading-h2" id="search-result-heading__text" tabindex="-1">{{ (caseSt
             <input class="govuk-checkboxes__input" id="select-{{ result.case_id }}" name="select-{{ result.case_id }}"
               type="checkbox" (change)="changeSelection(result)" [checked]="isSelected(result)" [disabled]="!canBeShared(result)" />
             <label class="govuk-label govuk-checkboxes__label" for="select-{{ result.case_id }}">
+              Select Case Reference {{ result.case_id }}
```

Comment:
This should be "Select case" not "Select Case Reference". You are not selecting the reference, you are selecting the case, and it should be in sentence case.